samples/samples-aspnet/embedded-sign-in-widget/embedded-sign-in-widget/AuthenticationHelper.cs

```diff
@@ -64,6 +66,7 @@ public static async Task<SignInWidgetConfiguration> StartWidgetSignInAsync(this
                 signInWidgetConfiguration = widgetSignInResponse.SignInWidgetConfiguration;
             }
             session[idxContext.State] = idxContext;
+            session[IdxStateKey] = idxContext.State;
```

Comment:
It seems we're already storing the state as a key in line 68. I wonder if we need both entries, one as a key and another one as value?

Reply:
Line 68 = : ["randomstatevalue-12345"] = context;
LIne 69 = : ["IdxStateKey"] = "randomstatevalue-12345";

This is to enable retrieving the value using a known key.